snuba/cli/multistorage_consumer.py

```diff
@@ -213,6 +218,15 @@ def multistorage_consumer(
             consumer_configuration, producer=producer, commit_log_topic=commit_log,
         )
 
+    dead_letter_producer: Optional[KafkaProducer] = None
+    dead_letter_queue: Optional[Topic] = None
+    if dead_letter_topic:
+        dead_letter_queue = Topic(dead_letter_topic)
+
+        dead_letter_producer = KafkaProducer(
+            build_kafka_producer_configuration(StreamsTopic(dead_letter_topic))
```

Comment:
this didn't like the arroyo `Topic`, probs need to fix that

Reply:
What does this mean? It's not supposed to be the arroyo topic. It's the topic for fetching the configuration from settings in Snuba.

Reply:
I guess it was just confusing to have two separate topic objects